db/migrate/20180719162710_add_owner_and_group_to_auth.rb

```diff
@@ -0,0 +1,35 @@
+class AddOwnerAndGroupToAuth < ActiveRecord::Migration[5.0]
+  class Authentication < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+
+    include ReservedMigrationMixin
+    include MigrationStubHelper
+  end
+
+  def up
+    add_column :authentications, :evm_owner_id, :bigint
+    add_column :authentications, :miq_group_id, :bigint
+    add_column :authentications, :tenant_id, :bigint
+
+    say_with_time("Migrate evm_owner and miq_group from reserved table to authentications") do
+      Authentication.includes(:reserved_rec).each do |auth|
```

Comment:
TODO: thinking about using the userid string field to store the evm_owner_userid name, if so we will need to clear it here

Reply:
I don't think that's necessary because it will get cleared (return nil) when `evm_owner_id` is cleared